src/js/components/put-away/PutAwayCheckPage.jsx

```diff
@@ -216,13 +215,18 @@ class PutAwayCheckPage extends Component {
     }
   }
 
+  firstPage() {
```

Comment:
If you are using Wizard component, then you should use `goToPage` for handling this.